doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -231,6 +231,12 @@ Thingy:53: Matter weather station
   * The :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_ZAP_FILES_PATH` Kconfig option, which specifies ZAP files location for the application.
     By default, the option points to the :file:`src/default_zap` directory and can be changed to any path relative to application's location that contains the ZAP file and :file:`zap-generated` directory.
 
+Protocols serialization
+-----------------------
+
+* Added:
+  * Protocol serialization server application.
```

Comment:
```suggestion
Protocols serialization
-----------------------

* Added the protocol serialization server application.
```
Let's move it to after the nRF Machine Learning (Edge Impulse) section to keep it in alphabetical order

Reply:
nit: "Protocol" -> "Protocols" everywhere.

Reply:
fixed.

Reply:
fixed.